xrpl/wallet/main.py

```diff
@@ -16,21 +16,29 @@ class Wallet:
     details.
     """
 
-    def __init__(self: Wallet, seed: str, sequence: int) -> None:
+    def __init__(
+        self: Wallet,
+        seed: str,
+        sequence: int,
+        *,
```

Comment:
What does this `*` mean here?

Reply:
It means that you have to specify the keyword as a part of your argument, and you can't just add another param to the list